about_scoring_project.rb

```diff
@@ -30,7 +31,47 @@
 # Your goal is to write the score method.
 
 def score(dice)
-  # You need to write this method
+  result = 0
```

Comment:
I liked the approach, but there is an easier solution for this. please re-think the method.